go/service/ctl.go

```diff
@@ -25,14 +24,8 @@ func NewCtlHandler(xp rpc.Transporter, v *Service, g *libkb.GlobalContext) *CtlH
 	}
 }
 
-// Stop is called on the rpc keybase.1.ctl.stop, which shuts down the service.
-func (c *CtlHandler) Stop(_ context.Context, args keybase1.StopArg) error {
-	c.G().Log.Debug("Received stop(%d) RPC; shutting down", args.ExitCode)
-	go func() {
-		c.service.Stop(args.ExitCode)
-	}()
-	return nil
-}
+// Stop is located in ctl_osx.go and ctl_non_osx.go for
```

Comment:
Instead, can you check `Service.ForkType == keybase1.ForkType_LAUNCHD`?  Then you shouldn't need an OSX and a non-OSX file.